src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+  onTabSelected: (tab: Tab) => void;
+  selectedTabId: string;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+}) => {
+  const isSelectedTab = tabs.find(tab => selectedTabId === tab.id)
+  || tabs[0];
```

Comment:
So when you'll fix the comment above, you'll have the possibility to destructure the selected tab here